server/requirements.txt

```diff
@@ -9,3 +9,5 @@ django-allauth
 celery<4.0
 django-celery
 django-filter
+pytest
+pytest-django
```

Comment:
Maybe have requirements.txt in ./tests and separate those packages from the server ?